app/components/search-result.js

```diff
@@ -8,7 +8,9 @@ export default Ember.Component.extend({
         PeerJ: 'peerj',
         arXiv: 'arxiv'
     },
-
+    didRender() {
+        MathJax.Hub.Queue(['Typeset', MathJax.Hub]);  // jshint ignore:line
```

Comment:
My understanding is that usually, there is another argument here. By omitting it, we may be causing mathjax to do far more work than is necessary.

http://docs.mathjax.org/en/latest/api/hub.html#Typeset
> Calls the preprocessors on the given element (or elements if it is an array of elements), and then typesets any math elements within the element. **If no element is provided, the whole document is processed.**

Since this is a component, I would recommend searching for the relevant text area you want to process within that component. Ember provides a jquery selector relative to the root of the given component instance, for example, via `this.$.()`.